src/embed.ts

```diff
@@ -284,9 +284,10 @@ async function getWikimediaThumbnail(url: URL) {
 
   // note: this adds manual caching settings,
   // since wikimedia does not return a max - age setting that cf can adopt
+  // TODO: disable `cacheEverything` when blocked image is working again
   const imgRes = await fetch(previewImageUrl, {
-    cf: { cacheTtl: 24 * 60 * 60, cacheEveything: true }, // TODO: disable `cacheEverything` when blocked image is working again
```

Comment:
why did you decide to leave `cacheEverything` set to true?

Reply:
That's the code for the wikimedia thumbnail. Currently we hit too many requests to Wikimedia and get a 429er response. To reduce our number of requests to wikimedia we need to cache also those 429er responses so that we do no query against the wikimedia servers so often...

Reply:
Changes were made to fix https://github.com/serlo/serlo.org-cloudflare-worker/issues/213

Reply:
I know, but this was quite some time ago. If this is still current, the `cacheEverything` is not working and we need to think of smth. else.

Imho `cacheEverything` does not make sense in the long run, so I only disagree with removing the comment.

Reply:
Look: We had an error all the time, thus it couldn't work. `cacheEverything` was spelled wrong. See https://serlo.slack.com/archives/CEB781NCU/p1632824890004800